RUFAS/biophysical/manure/handler/handler.py

```diff
@@ -0,0 +1,255 @@
+from dataclasses import dataclass
+from typing import Any
+
+from numpy import clip
+
+from RUFAS.biophysical.manure.processor import Processor
+from RUFAS.current_day_conditions import CurrentDayConditions
+from RUFAS.data_structures.animal_to_manure_connection import ManureStream
+from RUFAS.general_constants import GeneralConstants
+from RUFAS.time import Time
+from RUFAS.units import MeasurementUnits
+
+
+@dataclass(kw_only=True)
+class HandlerConfig:
+    """
+    Class for storing the configuration of a manure handler.
+
+    Attribute
+    ----------
+    name : str
+        The name of the manure handler.
+    manure_handler_type: str
+        The class of manure handlers that this configuration falls into.
+    cleaning_water_use_amount : float
+        Amount of cleaning water used per animal per day (L).
+    minutes_per_cleaning : int
+        Number of minutes needed per animal per cleaning, minutes.
+    cleanings_per_day : int
+        Number of cleanings per day.
+    cleaning_water_recycle_fraction : float
+        Fraction of cleaning water that is from recycled (not fresh) water sources.
+    use_parlor_flush : bool
+        Indication for if a parlor flush is used in addition to routine parlor water cleaning with fresh water.
+
+    """
+
+    name: str
+    manure_handler_type: str
+    cleaning_water_use_amount: float
+    minutes_per_cleaning: int
+    cleanings_per_day: int
+    cleaning_water_recycle_fraction: float
+    use_parlor_flush: bool
+
+
+class Handler(Processor):
+    """
+    Base class for all handlers.
+
+    Parameters
+    ----------
+    name : str
+        Unique identifier of the processor.
+    is_housing_emissions_calculator : bool
+        Indicates if a Processor calculates housing emissions.
+
+    Attributes
+    ----------
+    manure_stream : ManureStream
+        The ManureStream instance being checked for compatibility.
+    fresh_water_volume_used_for_milking : float
+        The amount of fresh water used for milking (L).
+    ammonia_emission : float
+        The amount of ammonia emission (kg).
+    config : HandlerConfig
+        The custom configuration for the handlers.
+
+    """
+
+    def __init__(self, name: str, is_housing_emissions_calculator: bool, config: HandlerConfig):
+        super().__init__(name, is_housing_emissions_calculator)
```

Comment:
```suggestion
    def __init__(self, name: str, config: HandlerConfig):
        super().__init__(name, is_housing_emissions_calculator=True)
```
For now, I think it is safe to assume that all handlers will be considered housing emissions calculators.